### DISCUSSION TITLE
Can I trigger onboarding Renovate via topics?

### DISCUSSION THREAD

**Discussion by ysenturk1** [renovatebot/renovate#19408] [Request Help]
Hello,

I would like to trigger renovate onboarding via GitHub topics, for example if our Devs add 'renovate' tag into their repo I want Renovate to trigger the onboarding instead of adding it into every repo in the org or adding the repos manually. Is this possible? Sorry if this has asked before.

**Comment by viceice** [comment]
not possible

**Comment by rarkins** [comment]
How are you running Renovate? As an app, or using a personal access token?

**Comment by ysenturk1** [comment]
We are hosting the app in our Github Enterprise and using Renovate On-prem.

**Reply by rarkins** [reply]
So your goal is to install into "All repositories" but only run on those which add a topic?

**Reply by ysenturk1** [reply]
Exactly!

**Reply by rarkins** [reply]
Understood, but this functionality is not currently planned

**Comment by secustor** [comment]
See this issue and the comments on it. Their you find a solution for you. https://github.com/renovatebot/renovate/issues/10011#issuecomment-992568583

**Reply by secustor** [reply]
Ah, seeing right now that you are using renovate on-prem. In that case your milage may vary.
